bridges/centralized-ethereum/src/actors/dr_sender/tests.rs

```diff
@@ -53,7 +53,7 @@ fn deserialize_dr_high_value() {
     // Setting the maximum allowed value to 1 nanowit below that will result in an error 0xE1:
     // BridgePoorIncentives
     let err = deserialize_and_validate_dr_bytes(&dro_bytes, total_value - 1).unwrap_err();
-    assert_eq!(err.encode_cbor(), vec![216, 39, 129, 24, 225]);
```

Comment:
This shouldn't change, I think it's because the request created above is no longer valid. Maybe `example_request` is setting the kind to unknown instead of http-get?